### PR TITLE
Information on the gas cost of operating the client

### PR DESCRIPTION
**Pull request by pdyraga**
Added information about gas costs of operations the beacon client is performing. It is important that the operators understand the cost of running the client and can estimate the required ETH based on their stake, current gas price, and the expected traffic in the network.

### CODE REVIEW

File: docs/run-random-beacon.adoc [hunk 1]
```diff
@@ -43,6 +43,35 @@ Keep Random Beacon client.
 |2 vCPU / 4 GiB RAM / 1 GiB Persistent Storage
 |===
 
+== Gas Costs
+
+Random Beacon smart contracts reimburse the operator for successfully submitting relay entry and DKG result but they do not reimburse for submitting a group selection ticket. Reimbursements are sent to the beneficiary account and can be claimed along with rewards once the group expires. It is expected that the operators have enough ETH on the accounts used by clients to submit the required transactions and that the operator account balance is monitored and refilled as needed. Bear in mind that the higher stake is, the operator is selected more frequently and is expected to submit more transactions as a result.
```
Comment (Shadowfiend) [inline docs/run-random-beacon.adoc:48]: Let's line wrap here and in the next paragraph.

File: docs/run-random-beacon.adoc [hunk 2]
```diff
@@ -43,6 +43,35 @@ Keep Random Beacon client.
...
+|Gas Cost
+
+|Submit group selection ticket
+|140 000
+
+|Submit DKG result
+|1 740 000
+
+|Submit relay entry
+|280 000
```
Comment (Shadowfiend) [inline docs/run-random-beacon.adoc:64]: Can we add a column on reimbursed yes/no?

File: docs/run-random-beacon.adoc (hunk 1 above)
Comment (Shadowfiend) [reply docs/run-random-beacon.adoc:48]: Maybe we can also clarify (could be its own paragraph after the table, not sure) that reimbursements happen after successful group formation?

Comment (pdyraga) [reply docs/run-random-beacon.adoc:48]: > Maybe we can also clarify (could be its own paragraph after the table, not sure) that reimbursements happen after successful group formation?

We do have:

> Random Beacon smart contracts reimburse the operator for successfully submitting relay entry and DKG result

🤔 Isn't it clear from this sentence?

**Comment by pdyraga** [comment]
It's ready for another chance @Shadowfiend

**Review by Shadowfiend — APPROVED** [review]
Yah let's do it.

**Pull request merged**